src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -538,6 +538,40 @@ namespace OpenMS
     // Algorithm
     // ################
 
+    void SiriusAdapterAlgorithm::logInSiriusAccount(String& executable, const String& email, const String& password) const
+    {
+      if (!email.empty() && !password.empty())
+      {
+        // sirius login --email=email --password=password
+        QString executable_qstring = SiriusAdapterAlgorithm::determineSiriusExecutable(executable).toQString();
+        QStringList command_line{"login", String("--email="+email).toQString(), String("--password="+password).toQString()};
+        
+        // start QProcess with sirius login command
+        QProcess qp;
```

Comment:
Do we have a command in OpenMS that encapsulates the qprocess?

Reply:
No, only in TOPPBase